packages/core/src/server/cms/plp.ts

```diff
@@ -65,10 +64,6 @@ const getPLPFromCmsEnvData = async (
 ): Promise<PLPContentType> => {
   const pages: PLPfromCmsEnvData[] = allPLPsFromCMSData ?? []
 
-  if (!pages[0]) {
-    throw new MissingContentError(options)
-  }
```

Comment:
question: why was it removed?

Reply:
I don't remember hahah
but I tested it again with this code and it worked as expected. So I added it back [here](https://github.com/vtex/faststore/pull/2668/commits/a67d01e1d7fea1cbcbd824cfb8bd13510f1669d5).
nice catch!